region/maridia/inner-yellow.json

```diff
@@ -1946,7 +1983,27 @@
               ],
               "openEnd": 1
             }
-          ]
+          ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get Hit By Zoa",
+                  "notable": false,
+                  "requires": []
```

Comment:
I think this has an implied h_canCrouchJumpDownGrab 
or 
jump into room and this room has normal physics.

Because you fall in the water entering this room.